mlir-tensorrt/executor/lib/Runtime/Backend/Lua/LuaRuntime.cpp

```diff
@@ -650,9 +650,6 @@ parseResults(const sol::protected_function_result &pfr,
     if (!memref.isOk())
       return memref.getStatus();
 
-    // Increment external reference count since we are returning a memref
-    allocator.incrementExternalCount(info.ptr);
```

Comment:
We do not need this. Reference count should only be incremented if a explicitly create a DL pack tensor on a memref. 
Here memref is internally owned and tracked.